test/vm/kernels.jsonl

```diff
@@ -0,0 +1,11 @@
+{"headers": ["http://vault.centos.org/centos/6.9/os/x86_64/Packages/kernel-devel-2.6.32-696.el6.x86_64.rpm"], "kernels": ["http://vault.centos.org/centos/6.9/os/x86_64/Packages/kernel-2.6.32-696.el6.x86_64.rpm"]}
+{"headers": ["http://mirror.centos.org/centos/7.9.2009/updates/x86_64/Packages/kernel-devel-3.10.0-1160.49.1.el7.x86_64.rpm"], "kernels": ["http://mirror.centos.org/centos/7.9.2009/updates/x86_64/Packages/kernel-3.10.0-1160.49.1.el7.x86_64.rpm"]}
+{"headers": ["http://amazonlinux.us-east-1.amazonaws.com/2/core/2.0/x86_64/e5756b881dd10f379f337f77c404dbc4c557dcbacab9b1b8a3aea0fb859ea0fb/../../../../../blobstore/635c3af49a948c3844fc8b638ef25898aeeb863ea647a417e6e8a4f2d58fc231/kernel-devel-4.14.296-222.539.amzn2.x86_64.rpm"], "kernels": ["http://amazonlinux.us-east-1.amazonaws.com/2/core/2.0/x86_64/e5756b881dd10f379f337f77c404dbc4c557dcbacab9b1b8a3aea0fb859ea0fb/../../../../../blobstore/0510a899162b40d750092d0750b0f25966a42c1bc435bcb3a3c0a67d16d8e5ca/kernel-4.14.296-222.539.amzn2.x86_64.rpm"]}
+{"headers": ["https://mirror.rcg.sfu.ca/mirror/ELRepo/kernel/el7/x86_64/RPMS/kernel-lt-devel-5.4.231-1.el7.elrepo.x86_64.rpm"], "kernels": ["https://mirror.rcg.sfu.ca/mirror/ELRepo/kernel/el7/x86_64/RPMS/kernel-lt-5.4.231-1.el7.elrepo.x86_64.rpm"]}
```

Comment:
I've found missing Centos kernel headers packages, from mirrors:

- https://mirror.rcg.sfu.ca/mirror/ELRepo/kernel/el7/x86_64/RPMS/kernel-lt-devel-5.4.231-1.el7.elrepo.x86_64.rpm -> 503
- https://mirror.rcg.sfu.ca/mirror/ELRepo/kernel/el7/x86_64/RPMS/kernel-ml-devel-5.10.16-1.el7.elrepo.x86_64.rpm
- -> 503
- https://mirror.rcg.sfu.ca/mirror/ELRepo/kernel/el7/x86_64/RPMS/kernel-ml-devel-5.14.15-1.el7.elrepo.x86_64.rpm -> 503
- http://mirrors.coreix.net/elrepo/kernel/el7/x86_64/RPMS/kernel-ml-devel-6.3.4-1.el7.elrepo.x86_64.rpm -> 404

detailed log:
```
--2023-06-17 13:38:13--  https://mirror.rcg.sfu.ca/mirror/ELRepo/kernel/el7/x86_64/RPMS/kernel-lt-devel-5.4.231-1.el7.elrepo.x86_64.rpm
Resolving mirror.rcg.sfu.ca (mirror.rcg.sfu.ca)... 142.58.94.6
Connecting to mirror.rcg.sfu.ca (mirror.rcg.sfu.ca)|142.58.94.6|:443... connected.
HTTP request sent, awaiting response... 503 Service Unavailable
2023-06-17 13:38:14 ERROR 503: Service Unavailable.

--2023-06-17 13:38:14--  https://mirror.rcg.sfu.ca/mirror/ELRepo/kernel/el7/x86_64/RPMS/kernel-ml-devel-5.10.16-1.el7.elrepo.x86_64.rpm
Reusing existing connection to mirror.rcg.sfu.ca:443.
HTTP request sent, awaiting response... 503 Service Unavailable
2023-06-17 13:38:14 ERROR 503: Service Unavailable.

--2023-06-17 13:38:14--  https://mirror.rcg.sfu.ca/mirror/ELRepo/kernel/el7/x86_64/RPMS/kernel-ml-devel-5.14.15-1.el7.elrepo.x86_64.rpm
Reusing existing connection to mirror.rcg.sfu.ca:443.
HTTP request sent, awaiting response... 503 Service Unavailable
2023-06-17 13:38:14 ERROR 503: Service Unavailable.

--2023-06-17 13:38:14--  http://mirrors.coreix.net/elrepo/kernel/el7/x86_64/RPMS/kernel-ml-devel-6.3.4-1.el7.elrepo.x86_64.rpm
Resolving mirrors.coreix.net (mirrors.coreix.net)... 85.13.241.50, 2a01:c0:2:3d::2
Connecting to mirrors.coreix.net (mirrors.coreix.net)|85.13.241.50|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-06-17 13:38:14 ERROR 404: Not Found.
```

Maybe we'd like to find alternative ones?

Reply:
Amazing, yes why would we ever something just a little bit stable ... adjusted it, but more importantly left some notes in the readme that mirrors could be unavailable or break over time ...